app/policies/template_policy.rb

```diff
@@ -1,76 +1,81 @@
 class TemplatePolicy < ApplicationPolicy
   attr_reader :user, :template
-
-  def initialize(user, template)
-    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+  
+  def initialize(user, template = Template.new)
+    raise Pundit::NotAuthorizedError, _("must be logged in") unless user.is_a?(User)
     @user = user
     @template = template
   end
-
-  ##
-  # Users can modify templates if:
-  #  - They can modify templates
-  #  - The template which they are modifying belongs to their org
-  ##
-
-  def admin_index?
+  
+  def index?
     user.can_modify_templates?
   end
 
-  def admin_template?
-    user.can_modify_templates?  &&  (template.org_id == user.org_id)
+  def new?
+    user.can_modify_templates?
   end
 
-  def admin_customize?
+  def create?
     user.can_modify_templates?
   end
 
-  def admin_publish?
-    user.can_modify_templates?  &&  (template.org_id == user.org_id)
+  def edit?
+    user.can_super_admin? || (user.can_modify_templates? && template.org_id == user.org_id)
   end
 
-  def admin_unpublish?
-    user.can_modify_templates?  &&  (template.org_id == user.org_id)
+  def update?
+    user.can_super_admin? || (user.can_modify_templates? && template.org_id == user.org_id)
   end
 
-  def admin_update?
-    user.can_modify_templates?  &&  (template.org_id == user.org_id)
+  def destroy?
+    user.can_super_admin? || (user.can_modify_templates?  &&  (template.org_id == user.org_id))
   end
-
-  def admin_new?
-    user.can_modify_templates?
+  
+  def history?
+    user.can_super_admin? || (user.can_modify_templates? && template.org_id == user.org_id)
   end
 
-  def admin_create?
-    user.can_modify_templates? && (template.org_id.nil? || (template.org_id == user.org_id))
+  def customize?
```

Comment:
super_admins cannot modify templates with current auth system. 

Dont think these display for them on the front end either?? so this may not be an issue

Reply:
Concern covered by another ticket